zcash_client_backend/src/keys.rs

```diff
@@ -74,12 +86,56 @@ pub enum DerivationError {
     Transparent(hdwallet::error::Error),
 }
 
+/// A version identifier for the encoding of unified spending keys.
+///
+/// Each epoch corresponds to a range of block heights during which the unified spending key parsed
+/// from an encoded form tagged with that epoch's identifier is expected to provide sufficient
+/// spending authority to spend any non-Sprout shielded note created in a transaction within the
```

Comment:
This might not be a correct description in future if, e.g. we close a pool (after Sprout) for spending. I don't know how important it is to document that now. (Non-blocking.)